stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java

```diff
@@ -156,6 +180,15 @@ public ClientIOTaskResult runLocal() throws Exception {
     return taskResult;
   }
 
+  private BenchThread getBenchThread(BenchContext context, int index) {
+    if (mParameters.mClientType == FileSystemClientType.ALLUXIO_HDFS) {
+      return new AlluxioHDFSBenchThread(context, mCachedFs[index % mCachedFs.length], index);
+    }
+
+    return new AlluxioNativeBenchThread(context,
```

Comment:
nit: You might do a `switch` and throw error if the type is not handled. Not a must.